src/control/client/config_test.go

```diff
@@ -63,33 +78,32 @@ func TestLoadConfigDefaultsNoFile(t *testing.T) {
 	log, buf := logging.NewTestLogger(t.Name())
 	defer common.ShowBufferOnFailure(t, buf)
 
-	defaultConfig := getDefaultConfig(t)
+	defaultConfig, f, closure := getDefaultConfig(t, log)
+	f.Close()
+	defer closure()
```

Comment:
nit: Naming it closure() isn't helpful for the reader. Yes, it is a closure, but what does it do? A better name/convention would be to call it cleanup() or otherwise allow the reader to understand what it's for without having to go look at the source of getDefaultConfig()